src/shared/messageBundles.js

```diff
@@ -167,24 +165,28 @@ gpii.app.messageBundles.getMessageDistributorGrade = function (messageBundles, d
 
 /**
  * Loads synchronously and parses the messageBundles file.
- * @param messageBundlesPath {String} The path to the messageBundles file relative
+ * @param {String} messageBundlesPath - The path to the messageBundles file relative
  * to the project directory.
  * @return {Object} The parsed message bundles for the different locales.
  */
 gpii.app.messageBundles.loadMessageBundles = function (messageBundlesPath) {
-    var resolvedPath = require("path").resolve(messageBundlesPath);
+    if (fluid.require) {
+        return fluid.require("%gpii-app/" + messageBundlesPath);
+    }
+    // Required by renderer process
+    var resolvedPath = require("path").join(__dirname, "../../..", messageBundlesPath);
```

Comment:
The fact that this line shows up with no coverage suggested we may have some problem with collecting coverage data from electron renderer processes - @georgitodorov , could you verify this is really called and perhaps @the-t-in-rtf  can comment

Reply:
Right, currently the coverage data is only for the main process.